src/models.py

```diff
@@ -1,13 +1,14 @@
 import numpy as np
 import torch.optim as optim
 
-from steps.pytorch.architectures.unet import UNet, UNetMultitask
-from steps.pytorch.callbacks import CallbackList, TrainingMonitor, ValidationMonitor, ModelCheckpoint, \
+from .steppy.pytorch.architectures.unet import UNet, UNetMultitask
```

Comment:
@taraspiotr I think we will go with Resnet101/152 so not sure how usefull are those 2